frontend/public/locales/en/translation.json

```diff
@@ -1948,6 +1949,7 @@
     "Show only:": "Show only:",
     "show.less": "Show less",
     "show.more": "{{number}} more",
+    "Some policies have a Prune Object Behavior set": "Some policies have a Prune Object Behavior set",
```

Comment:
Some policies have the `prune` parameter set.

Not sure what we might agree for the parameter name

Reply:
I agree, @heyethankim, do you have any new thoughts on the name?

Reply:
Well, I like your name suggestion. 😀 Would you like to bring it up during the GRC security meeting on Monday?